src/views/undo.c

```diff
@@ -19,12 +19,23 @@
 #include "views/undo.h"
 #include <glib.h>    // for GList, gpointer, g_list_first, g_list_prepend
 #include <stdlib.h>  // for NULL, malloc, free
+#include <sys/time.h>
+
+const unsigned long long MAX_TIME_PERIOD = 500; // 500ms
+
+static unsigned long long _get_timestamp(void)
```

Comment:
Hum... embarrassing :) I did not notice this routine which seems to be what I need. I'll fix.